src/pcapi/routes/serialization/offerers_serialize.py

```diff
@@ -76,11 +79,14 @@ class GetOffererResponseModel(BaseModel):
     _humanize_id = humanize_field("id")
 
     @classmethod
-    def from_orm(cls, offerer: Offerer):  # type: ignore
+    def from_orm(cls, offerer: Offerer, venue_stats_by_ids: Optional[Dict[int, VenueStatsResponseModel]] = None):  # type: ignore
         offerer.apiKey = {
             "maxAllowed": settings.MAX_API_KEY_PER_OFFERER,
             "prefixes": get_api_key_prefixes(offerer.id),
         }
+        if venue_stats_by_ids:
+            for managedVenue in offerer.managedVenues:
```

Comment:
(détail / style) 

idem plus haut :

```suggestion
            for venue in offerer.managedVenues:
```